api/v1/helpers/verbs/utils.js

```diff
@@ -87,8 +94,22 @@ function handleUpdatePromise(resultObj, req, retVal, props, res) {
     redisCache.del(findCacheKey);
   }
 
+  resultObj.dbTime = new Date() - resultObj.reqStartTime;
+  logAPI(req, resultObj, returnObj);
+
   return res.status(constants.httpStatus.OK)
-    .json(responsify(retVal, props, req.method));
+    .json(responsify(returnObj, props, req.method));
+}
+
+/**
+ * @param {Array} arr Array of objects
+ * @param {String} fieldName The field to sort by
+ * @returns {Array} Array sorted by field
+ */
+function sortArrayObjectsByField(arr, fieldName) {
+  const arrCopy = JSON.parse(JSON.stringify(arr));
```

Comment:
Wondering why are we stringifying and parsing the json here?

Reply:
if we don't deep copy, the sort will be in-place. We do the deep copy to preserve the purity of the function (ie. not modify the params)